assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.stories.js

```diff
@@ -51,10 +51,16 @@ export default {
 	decorators: [
 		withQuery,
 		( Story, { parameters } ) => {
+			const setupRegistry = ( registry ) => {
+				if ( parameters?.setupRegistry ) {
+					parameters.setupRegistry( registry );
+				}
+			};
```

Comment:
Same as before, but also, there are no stories which include `setupRegistry` here, so this decorator should be something we can just remove.